index.html

```diff
@@ -3,20 +3,54 @@
   <head>
     <meta charset="utf-8">
     <title>ACLU Cash Bail</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/components/transition.min.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/components/dropdown.min.css">
```

Comment:
Do we need these? If we're writing our own styles for them can we avoid these imports, or are they doing other stuff we want?

Reply:
We can absolutely extract just the styles we need from these and then remove the imports. I'll leave a TODO here and do it as a follow up.